### DISCUSSION TITLE
Are there any plans to support Swift language?

### DISCUSSION THREAD

**Discussion by PublicWorld** [ccache/ccache#1029] [Questions & answers] [answered]
I am sorry that ccache can't be used in Swift. 
Do we have any plans to support it? Or is there any technical problem that is difficult to solve?

**Comment by jrosdahl** [comment, marked as answer]
There are no plans to support Swift. The compilation model is very different from the traditional C compiler model: https://forums.swift.org/t/ccache-and-or-distcc-for-swift/4117/4

**Reply by PublicWorld** [reply]
Oh~ got it, thank you!
